moveit_core/robot_trajectory/src/robot_trajectory.cpp

```diff
@@ -94,7 +94,7 @@ double RobotTrajectory::getAverageSegmentDuration() const
   if (duration_from_previous_.empty())
     return 0.0;
```

Comment:
```suggestion
  if (duration_from_previous_.size() <= 1)
  {
    return 0.0;
  }
```

Reply:
It's too bad we can't print a warning here, but it looks like there is no Node available. I don't want to break the API just for a warning.